test/unit/network_spec.js

```diff
@@ -67,11 +67,15 @@ describe('network', function() {
     // The test is valid for FF only: the XHR has support of the
     // 'moz-chunked-array' response type.
     // TODO enable for other browsers, e.g. when fetch/streams API is supported.
-    var m = /Mozilla\/5.0.*?rv:(\d+).*? Gecko/.exec(userAgent);
-    if (!m || m[1] < 9) {
-      expect(true).toEqual(true);
-      done();
-      return;
+    var isFirefoxWithMozChunkedEncodingSupport =
+      /Mozilla\/5.0.*?rv:(?:\d+).*? Gecko/.test(userAgent);
+    let isFetchWithStreamSupport = (typeof Response !== 'undefined' &&
+                                     'body' in Response.prototype &&
```

Comment:
There's one space too many on this line, please remove it.

Finally, you need to squash the commits when addressing review feedback; please see https://github.com/mozilla/pdf.js/wiki/Squashing-Commits.